Integrations/URS/reportscreen.xml

```diff
@@ -168,13 +168,13 @@
 
   <!-- Collapsable Row(s) area -->
   <Instance						Name="GroupInstance">
-    <Container				ID="Top"																					Size="1003,10"	AutoSize="V">
+    <Container				ID="Top"																					Size="1003,auto">
       <GridButton			ID="RowHeaderButton"								Offset="0,0"	Size="parent,30"		Style="RowButton"		String="$RowHeaderButton" >
         <Label ID="RowHeaderLabel" Anchor="R,C" Offset="50,0" Style="RowButton" String="Ends in ## turns (###)" Hidden="1" />
       </GridButton>
       <Image					ID="RowExpandCheck"		Anchor="R,T"	Offset="5,5"	Size="22,22"				Texture="Controls_ExpandButton" />
-      <ScrollPanel		ID="CollapseScroll"									Offset="0,31"	Size="parent,20"		StackGrowth="Bottom" FullClip="1" >
-        <SlideAnim		ID="CollapseAnim"			Start="0,0"	EndOffset="0,0" Speed="4" Cycle="Once" Stopped="1">
+      <ScrollPanel		ID="CollapseScroll"									Offset="0,31"	Size="parent,auto"	StackGrowth="Bottom" FullClip="1" >
```

Comment:
`parent,parent` instead of `parent,auto`, otherwise there's a bug when you collapse the rows.